internal/cli/ci.go

```diff
@@ -0,0 +1,91 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+
+	"github.com/windmilleng/tilt/internal/analytics"
+	"github.com/windmilleng/tilt/internal/cloud"
+	"github.com/windmilleng/tilt/internal/engine/buildcontrol"
+	"github.com/windmilleng/tilt/internal/store"
+	"github.com/windmilleng/tilt/internal/tiltfile"
+	"github.com/windmilleng/tilt/pkg/logger"
+)
+
+type ciCmd struct {
+	fileName             string
+	outputSnapshotOnExit string
+}
+
+func (c *ciCmd) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "ci [<tilt flags>] [-- <Tiltfile args>]",
+		DisableFlagsInUseLine: true,
+		Short:                 "Start Tilt in CI/batch mode with the given Tiltfile args",
+		Long: `
+Starts Tilt and runs services defined in the Tiltfile.
+
+Exits with failure if any build fails or any server crashes.
+`,
+	}
+
+	addWebServerFlags(cmd)
+
+	cmd.Flags().StringVar(&updateModeFlag, "update-mode", string(buildcontrol.UpdateModeAuto),
```

Comment:
do we expect this mode to run any updates after the initial deploy?

Reply:
good point, removed